app/controllers/accounts_controller.rb

```
@@ -1,11 +1,39 @@
 class AccountsController < ApplicationController
   def show
-    @account_page = AccountPage.new(find_subscribed_repos)
+    @account_page = AccountPage.new(
+      repos: find_subscribed_repos,
+      billable_email: current_user.billable_email
+    )
+  end
+
+  def update
+    customer = PaymentGatewayCustomer.new(current_user)
+
+    respond_to do |format|
+      format.json do
+        if customer.update_email(new_billable_email)
+          render json: updated_account_page
+        else
+          head :bad_gateway
```

What if they attempt to change to an email taken by another user? I'm not sure what Stripe does for that. Do they just accept it and send a confirmation, or will they fail fast?